src/Microsoft.DotNet.XHarness.CLI/CommandArguments/iOS/iOSTestCommandArguments.cs

```diff
@@ -99,6 +104,17 @@ public override IList<string> GetValidationErrors()
                 errors.Add($"Failed to find Xcode root at {XcodeRoot}");
             }
 
+            if (XmlResultJargon == XmlResultJargon.Missing)
+            {
+                var sb = new StringBuilder();
+                sb.AppendLine($"Failed to parse xml jargon. Available targets are:");
+                sb.AppendLine();
+                foreach (var val in new [] {XmlResultJargon.NUnitV2, XmlResultJargon.NUnitV3, XmlResultJargon.TouchUnit, XmlResultJargon.xUnit})
```

Comment:
We are doing this quite a lot around and could use some unified way of converting String->Enum->List of values. 
https://github.com/dotnet/xharness/blob/7ff61e57d730e9bd4da7849cece139ae92e5bbb1/src/Microsoft.DotNet.XHarness.CLI/Commands/iOS/iOSPackageCommand.cs#L276-L283

Leave it here as is, if we decide to do nullability, we will revisit these places anyway. But the future one would be nice to be case insensitive because I am quite struggling to type xUnit and NUnit correctly :)